### DISCUSSION TITLE
Send notification for non-KNX devices

### DISCUSSION THREAD

**Discussion by kevinkandziora** [brutella/hkknx-public#269] [Q&A] [answered]
Hi community,

As we know, it is possible to send notifications to the iPhone or other Apple devices when, for example, a window is open for too long. Now I was wondering if it is possible to send notifications when the status of a non-KNX device has changed. To make my example clearer, I will describe my scenario:
I have several _eve window & door smart contact sensors_ installed on my windows at home and I want _HomeKit_ or (_Home Configurator_) to send a notification when a window has been open too long. I hope this example is clear enough.
Maybe I'm answering my own question, but this means that hkknx has to read all devices from HomeKit and **not** from the installed KNX system, am I right?
I am currently using the app "_pushcut_" as a workaround, but I am not very happy with this solution. Does anyone have the same problem or a better idea to solve this problem?

kind regards.

**Comment by brutella** [comment, marked as answer]
Since hkknx doesn't have access to other HomeKit accessories, you have to use a workaround to get the behaviour that you want.

First you have to create an accessory in hkknx, with holds the state of one of your HomeKit sensor (ex. Eve contact sensor). You can use a switch for that. In hkknx you configure the switch to send the value `1` or `0` to a specific group address (ex. /1/2/3) when the state changes – use the field under Power → Write.

In HomeKit you create an automation to set the switch to On, when the contact sensor detects that no contact anymore.
In hkknx you can then create a script, which starts a timer and sends a notification when the value `1` is received from the group address 1/2/3.

**Reply by kevinkandziora** [reply]
I wasn't aware that this is possible. Thanks for the advice!
